test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsControllerSpec.scala

```diff
@@ -182,7 +182,13 @@ class ReliefDetailsControllerSpec
       }
 
     }
+    "handling check your answers" must {
+      def performAction(data: Seq[(String, String)]): Future[Result] =
+        controller.checkYourAnswers()(FakeRequest().withFormUrlEncodedBody(data: _*))
 
+      behave like residentsReliefSubmittedYesBehaviour(performAction, AmountInPence(500))
```

Comment:
you're only using these functions once - there's not much point in having them. If you want to see what the test is actually doing you have to go to the function which adds a level of indirection to the code

Reply:
Done, also I've noticed that the case residentsReliefSubmittedYesBehaviour was already being handled in the code. So removed it.